packages/api/src/external/carequality/process-outbound-patient-discovery-resps.ts

```diff
@@ -97,6 +106,12 @@ export async function processOutboundPatientDiscoveryResps({
         requestId,
         pdLinks: cqLinks.length,
         duration: elapsedTimeFromNow(discoveryParams.startedAt),
+        nonErrorCount,
+        errorCount,
+        httpErrorCount,
+        schemaErrorCount,
+        specificErrorCount,
```

Comment:
idea with sending all these different errorCount metrics to posthog is so we can track our overall error rate, and then if http Errors start spiking or schemaErrors start spiking, mitigate accordingly